src/pages/deti.tsx

```diff
@@ -125,7 +125,11 @@ const Deti: NextPage<Props> = ({
               </div>
             )}
 
-            <button className="govuk-button" type="submit">
+            <button
+              className="govuk-button"
+              type="submit"
+              disabled={values.children === undefined}
```

Comment:
Myslim ze spravne UX je nechavat submit button aktivny, ale zobrazit chyby ak na neho klikne a fields nie su validne

Reply:
Takto som to pochopil z designov vo figme. Ale mozno ma byt ten button iba sedy (resp vyzerat ako neaktivny a pritom klikatelny). Co myslis?

Reply:
Ja by som ich vobec nedaval disabled

Reply:
https://design-system.service.gov.uk/patterns/validation/
https://design-system.service.gov.uk/components/button/

```Disabled buttons

Disabled buttons have poor contrast and can confuse some users, so avoid them if possible.

Only use disabled buttons if research shows it makes the user interface easier to understand.```

Reply:
Dokonca podla toho prveho linku robime zle ze validujeme uz onBlur a nie az onSubmit 🤔